hx_lti_initializer/views.py

```diff
@@ -76,7 +77,13 @@ def initialize_lti_tool_provider(req):
 
 def create_new_user(username, user_id, roles):
     # now create the user and LTIProfile with the above information
-    user = User.objects.create_user(username, user_id)
+    # Max 30 length for person's name, do we want to change this? It's valid for HX but not ATG/FAS
```

Comment:
Is there any way to use the huid as the username, but send a different identifier to the CATCH? Couldn't you just send the ```user.id``` which is just an auto-generated number along with the user's full name? Is there any reason it has to send the username field? Does it display the username value? 

Alternatively, if it has to send the username field, what about using the base64 encoding of the sha1 hash of the huid ? SHA1 produces a hash of 160 bits, and since base64 encodes 6 bits at a time, that should work out to a hash of length 28. Something like this:

```python
import base64
import hashlib

example_huid = "12345678"
m = hashlib.sha1()
m.update(example_huid)
hashed_huid = base64.urlsafe_b64encode(m.digest())

print "Hash: ", hashed_huid, "\nLength: ", len(hashed_huid)
```

Output: ```Hash:  fCIvspJ9goryL1khNOiTJIBjfA0= Length:  28```

Although I'd probably be more in favor of increasing the size of the username field to accommodate the ```user_id``` than do the hashing above. Just some things to think about, for now. I won't hold up the PR, but this definitely needs to be worked out before we can release this as a pilot.